vulnerabilities/importer_yielder.py

```diff
@@ -227,13 +227,22 @@
         'data_source_cfg': {},
     },
     {
-    'name': 'elixir_security',
-    'license': '',
-    'last_run': None,
-    'data_source': 'ElixirSecurityDataSource',
-    'data_source_cfg': {
-        'repository_url': 'https://github.com/dependabot/elixir-security-advisories'
+        'name': 'elixir_security',
+        'license': '',
```

Comment:
They have specified the license at https://github.com/dependabot/elixir-security-advisories/blob/master/LICENSE.txt . Please run scancode-toolkit on that LICENSE and enter the license expression here.

Reply:
Ok